inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/openapi/DataProxyController.java

```diff
@@ -65,7 +65,7 @@ public Response<DataProxyConfig> getConfig(
             @RequestParam(required = true) String clusterName) {
         DataProxyConfig config = clusterService.getDataProxyConfig(clusterTag, clusterName);
         if (CollectionUtils.isEmpty(config.getMqClusterList()) || CollectionUtils.isEmpty(config.getTopicList())) {
-            return Response.fail("failed to get mq clusters or topics");
+            return Response.fail("Failed to get mq clusters or topics, make sure register mq to manager successfully");
```

Comment:
It seems not necessary to change this, because the topic list also may be empty.
So, check whether the MQ clusters are empty in the DataProxy is enough.

Reply:
make the message more meaningful.